test/parallel/test-dgram-oob-buffer.js

```diff
@@ -31,14 +31,14 @@ const socket = dgram.createSocket('udp4');
 const buf = Buffer.from([1, 2, 3, 4]);
 const portGetter = dgram.createSocket('udp4')
   .bind(0, 'localhost', common.mustCall(() => {
-    const address = portGetter.address();
+    const { address, port } = portGetter.address();
     portGetter.close(common.mustCall(() => {
-      socket.send(buf, 0, 0, address.port, address.address, common.noop);
-      socket.send(buf, 0, 4, address.port, address.address, common.noop);
-      socket.send(buf, 1, 3, address.port, address.address, common.noop);
-      socket.send(buf, 3, 1, address.port, address.address, common.noop);
+      socket.send(buf, 0, 0, port, address, common.mustNotCall());
```

Comment:
Looking at the original test at https://github.com/nodejs/node/commit/defa637378a it seems like this test may no longer have the same meaning originally intended. It seems to me that the key point was the assert.throws() calls. However, they no longer throw.